packages/amplication-server/src/core/ai/ai.service.ts

```diff
@@ -222,7 +227,7 @@ export class AiService {
               name: resource.name,
```

Comment:
Why do you need the where filter for the resource entities?

Reply:
Where do you mean?